src/components/WorkOrder/Photos/hooks/uploadFiles/fileUploadStatusLogger.ts

```diff
@@ -3,14 +3,19 @@ const fileUploadStatusLogger = (
   logCallback: (message: string) => void
 ) => {
   // initiate upload
-  logCallback(`Uploading ${totalFiles} file(s)`)
+  logCallback(`Processing ${totalFiles} file(s)`)
 
+  let numberOfCompletedCompressions = 0
   let numberOfCompletedUploads = 0
 
   // increment total each time an upload is completed
-  const fileUploadCompleteCallback = () => {
-    numberOfCompletedUploads++
-    logCallback(`${numberOfCompletedUploads} of ${totalFiles} file(s) uploaded`)
+  const fileUploadCompleteCallback = (action: 'Upload' | 'Compress') => {
+    if (action == 'Compress') numberOfCompletedCompressions++
+    if (action == 'Upload') numberOfCompletedUploads++
+    logCallback(
+      `Processing file(s): ${numberOfCompletedCompressions} of ${totalFiles} Compressed | ` +
+        `${numberOfCompletedUploads} of ${totalFiles} Uploaded`
+    )
   }
```

Comment:
Because compression is done before upload for all the files, showing the user the compression progress will help them know that the page is not frozen especially if it takes a while